test/functional/test_framework/test_node.py

```diff
@@ -37,7 +40,6 @@
     wait_until_helper,
     p2p_port,
     get_chain_folder,
-    EncodeDecimal,
```

Comment:
28011: as @coderabbitai discovered it in https://github.com/dashpay/dash/pull/6827#pullrequestreview-3142178017, we should also remove `EncodeDecimal` from `test/functional/test_framework/util.py` since it's no longer used anywhere.